dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/Application/AdaptiveCards.cs

```diff
@@ -148,7 +148,7 @@ public AdaptiveCards(Application<TState, TTurnStateManager> app)
         /// <returns>The application for chaining purposes.</returns>
         public Application<TState, TTurnStateManager> OnActionExecute(string verb, ActionExecuteAdaptiveCardHandler<TState> handler)
         {
-            return OnActionExecute(new Regex($"^{verb}$"), handler);
```

Comment:
how about just using `string.Equal()`?
does `Escape` correctly handle line breakers without `Singleline` or `Multiline` option?

Reply:
Oh I see, this is not due to `Escape` but because `$` allows a trailing newline `\n` character...
You're right, regex is error-prone, let me use `string.Equal` instead.